lib/longtail/mdnjs/toxml.py

```diff
@@ -0,0 +1,35 @@
+"""
+ Generate a longtail XML dump of the Fathead output.txt
+"""
+import csv
+
+from parse import FatWriter
+
+XML = """<?xml version="1.0" encoding="UTF-8"?>
+<add allowDups="true">
+{results}
+</add>
+"""
+
+DOC = """<doc>
+	<field name="title"><![CDATA[{title}]]></field>
+	<field name="paragraph"><![CDATA[{abstract} <a href="{source_url}">{source_url}</a>]]></field>
```

Comment:
commited changes adding l2_sec field to xml and json output

https://github.com/bfmags/zeroclickinfo-longtail/commit/4264b461cfc223e648859d7cffdb36b261d76ae1